src/db.py

```diff
@@ -160,3 +172,31 @@ def get_receipts(self):
                 _logger.exception(error)
             json["data"].append(element)
         return json
+
+    def add_email_for_sending(self, email: str):
+        session = self._sm()
+        c = session.query(MailList.id).filter(MailList.email == email).count()
+        if c > 0:
+            _logger.warning("E-mail already exists in database. STATUS_MAIL_ALREADY_EXIST")
+            return STATUS_MAIL_ALREADY_EXIST
+        new_email = MailList(
+            email=email,
+            status="active",
+        )
+        session.add(new_email)
+        session.commit()
+        session.refresh(new_email)
+        id = new_email.id
+        session.close()
+        if id is not None:
+            _logger.error("E-mail has been added successfully. STATUS_OK")
+            return STATUS_OK
+        else:
+            _logger.error("E-mail has not been added. STATUS_FAIL")
+            return STATUS_FAIL
+
+    def get_email_list_for_sending(self):
+        session = self._sm()
+        data = session.query(MailList.email).all()
+        if data:
+            return [''.join(x) for x in data]
```

Comment:
What will happen if data is none? In this case the method returns None and exception will be raised on the next step.

Reply:
I would suggest to return empty list. It's a nice approach when function return one type of data only.